docs/release-notes/NuGet-6.0.md

```diff
@@ -320,7 +321,18 @@ Install the [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/down
 
 **[List of all issues fixed in this release - 6.0](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvNDMwMDQ)**
 
-**[List of commits in this release - 6.0](https://github.com/NuGet/NuGet.Client/compare/5.11.0.17...6.0.0.262)**
+**[List of commits in this release - 6.0.0](https://github.com/NuGet/NuGet.Client/compare/5.11.0.17...6.0.0.262)**
+
+## Summary: What's New in 6.0.1
+
+Note: As this version only contains changes in Visual Studio, NuGet.exe, packages on nuget.org, and the .NET SDK were not updated.
```

Comment:
Do we need this note?

It's a bit confusing with the oxford comma. Otherwise we can just clarify that these are VS only fixes and leave out the rest?

Reply:
lol, I prefer Oxford commas, and think it adds clarity to sentences.

But sure, if you think it's clear enough saying it's only in VS, without mentioning the rest, I'll change it.

Reply:
😂 I don't think those reading our notes will see the subtle difference of your Oxford comma usage. ;)

Reply:
To be fair, there should have been a full stop after Visual Studio. It was a mistake to make that a comma. I still stand by a comma between "package on nuget.org" and "and the .NET SDK" though 😛